css/style.css

```diff
@@ -92,3 +92,313 @@ nav {
 ul li {
   list-style-type: none;
 }
+
+.action-button {
+  display: flex;
+  justify-content: center;
+  margin-top: 64px;
+  margin-bottom: 22px;
+}
+
+.recent-work header {
+  display: grid;
+  justify-content: center;
+}
+
+.recent-work h2 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  text-align: center;
+  margin-left: 26px;
+  margin-right: 22px;
+  margin-top: 12px;
+}
+
+.indicator-line {
+  display: flex;
+  justify-content: center;
+  margin-top: 24px;
+  margin-bottom: 62px;
+}
+
+.card-area {
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  margin-left: 26px;
+  margin-right: 22px;
+  margin-top: 62px;
+}
```

Comment:
- [ ] I think you should consider using `display: flex `to layout the content of each. This gives you control to layout your content nicely and also makes your UI responsive 📱 .

Reply:
I implemented this